src/styles/mixins/_bootstrap-normalize-alt.scss

```diff
@@ -0,0 +1,426 @@
+// sass-lint:disable-all
```

Comment:
Zie comment in `dso.scss` over de `-alt` suffix.

Er klopt meer niet. De file heet nog steeds `bootstrap-normalize`, maar het is nu eigenlijk een `dso-normalize` geworden. Deze file staat ook in de `styles/` folder

Reply:
eens